docs/source/conf.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import guzzle_sphinx_theme
+
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'ale'
+copyright = '2018, Adam Paquette'
+author = 'Adam Paquette'
```

Comment:
Can we swap this to be: `copyright = Public Domain author = USGS, Astrogeology Science Center`

Reply:
:+1: